src/app/search/search.component.ts

```diff
@@ -410,6 +420,40 @@ export class SearchComponent implements OnInit, OnDestroy {
       .catch((error) => console.log(error));
   }
 
+  /**
+   * Updates the results table when there is no search term
+   * Queries for 100 tools and 100 workflows
```

Comment:
tiny comment fix, query size is actually based on the query, not hardcoded anymore (which is good)